extensions/cornerstone-dicom-rt/src/loadRTStruct.js

```diff
@@ -0,0 +1,318 @@
+import dcmjs from 'dcmjs';
+const { DicomMessage, DicomMetaDictionary } = dcmjs.data;
+const dicomlab2RGB = dcmjs.data.Colors.dicomlab2RGB;
+
+async function checkAndLoadContourData(instance, datasource) {
```

Comment:
Thank you for moving this out to a separate file and having both implementations - I think it will be quite useful to have both ways of doing it here.